protected/models/Disorder.php

```diff
@@ -50,6 +51,8 @@ class Disorder extends BaseActiveRecordVersioned
     public static $SNOMED_DIABETES_TYPE_I_SET = array(46635009, 420868002);
     public static $SNOMED_DIABETES_TYPE_II_SET = array(44054006, 422014003);
 
+    public $is_ophthalmic;
```

Comment:
Not sure how I feel about this. If it doesn't crash then I guess its OK, but the use of public attributes that are not database columns does concern me a little bit.

If the precedent has been established elsewhere, then feel free to ignore this comment.